pkg/block/validations_test.go

```diff
@@ -0,0 +1,36 @@
+package block_test
+
+import (
+	"context"
```

Comment:
`make validate-fmt` says `Your code formatting is according to gofmt standards`.

Am I missing something?

Reply:
It ignores it since this is a test file but goimports requires separating standard library from 3rd party imports (this will fail on non-test files).
We exclude test files from lint and fmt due to coding style failures we have there for testing reasons but we still would like to maintain some of the formatting conventions.
If you run `goimports` on the file you will see the errors

Reply:
If it's important - why not apply only `imports` validation on the test files?

Anyway, fixed.